src/tls13api.rs

```diff
@@ -1,11 +1,9 @@
 // Import hacspec and all needed definitions.
-use crate::tls13formats::*;
-use crate::tls13handshake::*;
-use crate::tls13record::*;
-use crate::tls13utils::*;
 use hacspec_cryptolib::*;
 use hacspec_lib::*;
 
+use crate::{tls13formats::*, tls13handshake::*, tls13record::*, tls13utils::*};
```

Comment:
This is currently not allowed in hacspec.

Reply:
Oh, I wasn't aware of this limitation. We could use `imports_granularity=Item` (or "module") instead. See [Configurations.md#imports_granularity](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#imports_granularity).

Reply:
rustfmt doesn't touch any of this by default. So there should be no need to do anything. (Just skip the config on CI)

Reply:
The granularity has to be glob! But you can't group them. I don't think rustfmt can handle that.
Import styles aren't really that important to me. Depending on the module I sometimes prefer one and sometimes another one 🤷🏻‍♂️

Reply:
Reformatted code without --config and removed --config from CI job.